src/GitExtensions.PluginTemplate/Plugin.cs

```diff
@@ -43,14 +43,21 @@ public override void Unregister(IGitUICommands gitUiCommands)
             
         }
 
+        ///// <summary>
+        ///// See: https://github.com/gitextensions/gitextensions.plugintemplate/wiki/GitPluginBase#public-virtual-ienumerableisetting-getsettings
+        ///// </summary>
+        //public override IEnumerable<ISetting> GetSettings()
+        //{
+        //    // Uncomment and fill if your plugin have settings that the user could configure
+        //}

```

Comment:
Instead of commenting out, should this be as follows?
```cs
public override IEnumerable<ISetting> GetSettings() => Array.Empty<ISetting>();
```

Reply:
That's mostly the code in the base class but I don't find it enough incentive.
I prefer the commented code mostly because of the comment.

Reply:
Then should we remove it?

Reply:
No, we must keep it because that's the best way to inform the developers how to add config. They just have to read the comments and uncomment if they want to add settings.

Reply:
Ok